Added further steps for installation, updated PHP dependency (#1308)
lansuite/lansuite: website/docs/getting-started/installation.md

Use a (S)FTP-client then to upload the whole folder to your webspace.
Ensure that file permissions are properly set so that the webserver user is able to write both to the file `/inc/base/config.php` and the folder `./ext_inc/` and it's contents.

> ### Create Database and databaase user

Reviewer: ```suggestion
### Create Database and database user
```

If not provided you need to create a database user and database for LanSuite to store its data in.
While it is possible to share a database with a different application it is not recommended to do so, for both usability and security reasons.
The database user should only have privileges on the lansuite database and should only be able to connect from the webserver host, in most cases "localhost".
At the current point in time there is no separation between installation/update privileges and privileges for normal operation.

> Thus all privilegs on the neew database (and only there!) need to be granted to the lansuite database user.

Reviewer: ```suggestion
Thus, all privileges on the new database (and only there!) need to be granted to the lansuite database user.
```

Native SQL-Statements as reference:
```
CREATE DATABASE lansuite;
CREATE USER 'lansuite'@'localhost' IDENTIFIED BY '<strongpasswordhere>';
GRANT ALL ON lansuite.* TO 'lansuite'@'localhost';
```
It is recommended to do this before continuing with the application installation as the credentials, database and server names will be needed during the Installation Wizzard phase.

## Installation

> Once the source code files are ready for use and the database has been set up you can start the installation wizzard by accessing the destination folder.

Reviewer: ```suggestion
Once the source code files are ready for use and the database has been set up you can start the installation wizard by accessing the destination folder.
```

If you end up with a blank page or a "page not found"-error then check if browsing to  e.g. `lansuite/index.php` works.
In that case the webserver is not configured to serve index.php by default and should be configured accordingly.
If that also does not help then check the server error logs as these will provide a hint at what went wrong.

> ### Installation Wizzard

Reviewer: ```suggestion
### Installation Wizard
```

> Once the intial page loads you should see the guided installation wizzard that should help to create the required database scheme, configure essential settings and a first user.

Reviewer: ```suggestion
Once the initial page loads you should see the guided installation wizard that should help to create the required database scheme, configure essential settings, and create a first user.
```
